x/ccv/consumer/keeper/relay.go

```diff
@@ -61,7 +61,7 @@ func (k Keeper) OnRecvVSCPacket(ctx sdk.Context, packet channeltypes.Packet, new
 
 	// set outstanding slashing flags to false
```

Comment:
The comment is misleading. Can you expend why it's okay to remove the outstanding downtime flag.